lib/grit_adapter/git_layer_grit.rb

```diff
@@ -225,6 +227,10 @@ def refs(options, prefix)
       def repo
         @repo ||= Grit::Repo.new(@git.git_dir)
       end
+
+      def apply_patch(ref, patch=nil)
```

Comment:
Looks like grit's call to `git-apply` will fail and the method will just return `false`:
```ruby
2.0.0-p481 :004 > r.git.apply_patch({}, 'master', nil)
 => false
```